ECMA2Yaml/ECMAHelper/YamlGenerators/SDPTOCGenerator.cs

```diff
@@ -24,7 +24,7 @@ public static TocRootViewModel Generate(ECMAStore store)
                     toc.Items.Add(GenerateTocItemForNamespace(ns));
                 }
             }
-
+            toc.Metadata[OPSMetadata.V3TOCSplitItemsBy] = nameof(TocItemViewModel.Name).ToLower();
```

Comment:
This is not a metadata, it is a node property.

```suggestion
            toc.SplitItemsBy = nameof(TocItemViewModel.Name).ToLower();
```

Reply:
`TocRootViewModel.Metadata` is a `[ExtensibleMember]`, so it does just that.